spring-kafka-test/src/main/java/org/springframework/kafka/test/rule/KafkaEmbedded.java

```diff
@@ -222,6 +224,36 @@ public void before() throws Exception { //NOSONAR
 		System.setProperty(SPRING_EMBEDDED_ZOOKEEPER_CONNECT, getZookeeperConnectionString());
 	}
 
+	public Properties createBrokerProperties(int i) {
+		if (this.clientVersion.startsWith("1.0.")) {
+			return TestUtils.createBrokerConfig(i, this.zkConnect, this.controlledShutdown,
+					true, this.kafkaPorts[i],
+					scala.Option.apply(null),
+					scala.Option.apply(null),
+					scala.Option.apply(null),
+					true, false, 0, false, 0, false, 0, scala.Option.apply(null), 1);
+		}
+		else {
+			try {
+				Method method = TestUtils.class.getDeclaredMethod("createBrokerConfig",
```

Comment:
Can't we cache this method once in the `static` block instead of reflection on each call ?